src/components/PreviewDraft/PreviewMarkup.tsx

```diff
@@ -18,18 +18,13 @@ export interface MarkupPreviewProps {
 }
 
 export const PreviewMarkup = ({ article, language }: MarkupPreviewProps) => {
-  const { t } = useTranslation();
   return (
-    <StyledPreviewWrapper>
-      <OneColumn>
-        <PreviewDraft
-          type="article"
-          draft={article}
-          language={language}
-          previewAlt
-          label={t("form.previewProductionArticle.article")}
-        />
-      </OneColumn>
-    </StyledPreviewWrapper>
+    <PageContent variant="article">
+      <PageContent variant="content" asChild>
```

Comment:
Kvifor har du to PageContent nesta her?

Reply:
Det er sånn artikler er bygget opp. Den ytre bestemmer padding rundt artikkelen, mens den andre styrer artikkel-innholdet.